.buildkite/x-pack/pipeline.xpack.filebeat.yml

```
@@ -212,6 +212,9 @@ steps:
         env:
           ASDF_TERRAFORM_VERSION: "1.9.3"
           ASDF_PYTHON_VERSION: "3.9.13" # Not needed by ECH tests, but needed by VM
+          EC_ENDPOINT: "https://api.staging.elastic-gov.com"
+          TF_VAR_ech_region: "us-gov-east-1"
```

I think this should be `TF_VAR_ess_region`?
```suggestion
          TF_VAR_ess_region: "us-gov-east-1"
```

---

nope, renamed it here to ech